src/Service/CurriculumInventory/VerificationPreviewBuilder.php

```diff
@@ -193,7 +195,11 @@ public function getProgramExpectationsMappedToPcrs(array $data): array
 
         $expectations = $dedupedExpectations;
 
-        array_multisort(array_column($expectations, 'title'), SORT_ASC, $expectations);
+        array_multisort(array_column($expectations, 'title_plain'), SORT_ASC, $expectations);
+
+        array_walk($expectations, function (&$expectation) {
```

Comment:
By reference? For shame!! 😆.

Reply:
doctor's orders https://www.php.net/manual/en/function.array-walk.php ;)